### ISSUE TITLE
[Question] Any idea when this is released on NPM?

### ISSUE THREAD

**Issue by jclaessens97** [tomchen/stack-icons#1]
I very much like this idea and it seems there is no one else doing it, but it would even be better if it would be available on NPM registry. I saw it's in the TODO in the readme but any idea on when this is going to be published? 😄 

Thanks!

**Comment by tomchen** [comment]
Thanks for your attention.

I'm currently on some other small open source npm ts packages including a lightweight and deno-compatible alternative of cosmiconfig (this could be used by the future npm stack-icons package), and a ts port of one of my own python libraries.

But yeah, I can see stack-icons has some potential, I'll prioritize this one. I'll probably release it within, let's say, one or two weeks.

**Comment by jclaessens97** [comment]
Alright sounds cool! If there's anything I can do to help, you can ping me.

**Comment by jclaessens97** [comment]
Any progress? 😊

**Comment by tomchen** [comment]
Yeah I've published the small [config package](https://github.com/tomchen/searchconfig) I mentioned as well as the [ts lib](https://github.com/tomchen/bdfparser-js) ported from my python code, to practice typescript a little bit.

I'm sure I'll release this before or on friday, the last day I promised (Jan 30 + 2 weeks). Actually, this friday is the lunar new year for a year of the bull, wouldn't it be a fantastic day to release a package? 😄

**Comment by jclaessens97** [comment]
Exactly! Perfect day to release a package 😏🙌

**Comment by tomchen** [comment]
Discord server https://discord.gg/gg5AxGvY join if you want

**Comment by tomchen** [comment]
Happy lunar new year!

> Exactly! Perfect day to release a package 😏🙌
> Question was just informative, didn't meant to rush you or anything btw 😅

Don't worry, I have a secret script:
```bash
mkdir stackicons
cd stackicons
npm init -y
npm publish
```

Oops, I accidentally executed this script. Job done, problem solved, promise kept, issue closed. If you don't mind the unusability and emptiness of the package, voilà the completely blank [stackicons](https://www.npmjs.com/package/stackicons) 🤣🤣

---

Seriously, I'm renaming it to **geticon** and moving it to https://github.com/get-icon/geticon.

Because the name "stackicons" is not only taken by [a project](http://stackicons.com/) of the similar type (albeit inactive since 2014), its author seemed to have a [for-profit company](https://www.crunchbase.com/organization/stackicons) for it *(he could send me a request if he wants the name "stackicons" on npm & also on [twitter](https://twitter.com/stackicons) 🙂)*. So I'm gonna rename mine to avoid any future naming issue. My free open source not-for-profit project aims to provide a easy way for users to get one or several of those icons, so "geticon" seems to be a straightforward name. I was able to register the domain name [geticon.org](https://geticon.org/) (of course .com is unavailable) that was available but npm "geticon" is not, because "get-icon" is taken 4 years ago. I even wrote an email to get-icon's author, I'm still waiting for the reply but chance's low, even if his email is reachable and he agrees to cede the name, he could have easily lost his npm credential... I guess I'm gonna use scoped names like `npm i @geticon/core`, as I was able to grab the scope org account [geticon](https://www.npmjs.com/org/geticon) on npm...





For more details and plans, I could discuss on Discord.
